_data/coderkoala/coderkoala.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Nobel Dahal
+institution: Himalaya College of Engineering
+profile_pic: HIMALAYA071BCT029.jpg
```

Comment:
Incorrect file name, please do one of the following:
1. Change `HIMALAYA071BCT029.jpg` to `HCE071BCT029.jpg` in the markdown file.
2. Rename `HCE071BCT029.jpg` to `HIMALAYA071BCT029.jpg`.